apps/web/src/state/live/index.ts

```diff
@@ -22,10 +24,13 @@ export const initializeLiveState = async () => {
 	const publicKeys = publicKeyResponse.docs as PublicKeyDocument[];
 	const tokenResponse = await modules.storage.find(selectors.allTokens);
 	const tokens = tokenResponse.docs as TokenDocument[];
+	const settingsResponse = await modules.storage.find(selectors.settings);
```

Comment:
don't use `find`, use `modules.storage.get()` we need one single item, and we do know the id of it.